src/client/common/constants.ts

```diff
@@ -11,10 +11,7 @@ export const PYTHON_ALLFILES = [{ language: PYTHON_LANGUAGE }];
 export const GITHUB_ISSUE_MARKDOWN_FILE = [{ language: MARKDOWN_LANGUAGE, scheme: 'untitled', pattern: '**/issue.md' }];
 
 export const InteractiveInputScheme = 'vscode-interactive-input';
-export const NOTEBOOK_SELECTOR = [
-    { scheme: NotebookCellScheme, language: PYTHON_LANGUAGE },
-    { scheme: InteractiveInputScheme, language: PYTHON_LANGUAGE }
-];
+export const NOTEBOOK_SELECTOR = [{ language: PYTHON_LANGUAGE, notebookType: 'jupyter-notebook' }];
```

Comment:
Dont we need the Interactive Notebook type?

Reply:
Tried that, doesn't seem to fix it. Having trouble getting VSCode-OSS to work with interactive window.

Reply:
Trying that now. It looks like the algorithm for scoring changed. If the candidate URI doesn't match the filter at all, it exits. But if it does match it continues to look for a better score.

Reply:
Whereas before it would exit as soon as it found a match